src/Parser/Priest/Discipline/Modules/Features/AtonementHealingBreakdown.js

```diff
@@ -82,9 +82,42 @@ class AtonementHealingBreakdown extends React.Component {
                     />
                   </td>
                 </tr>
-              );
-            })}
-        </tbody>
+
+              , (bolts && bolts.map((value,index) => {
+
+                const penanceBarPercentage = value / healing.effective;
+
+                if(!value) return null;
+
+                return (<tr>
```

Comment:
I think it's more readable to start the element on the next line so that indentation is saner.